src/index.html

```diff
@@ -12,7 +13,40 @@
       href="styles/index.scss"
     />
```

Comment:
Link to a stylesheet with a `.scss` extension won't work correctly unless your server is configured to handle SCSS files directly, which is generally not the case. You should compile your SCSS to CSS and link to the compiled `.css` file instead.